Cargo.toml

```diff
@@ -18,10 +18,12 @@ license = "MIT OR Apache-2.0"
 readme = "README.md"
 
 [dependencies]
+# lazy-static (pulled in by e.g. nickel, regex) is incompatible with 1.22.1
+lazy_static = { version = ">=1.0,<1.2.0", optional = true }
```

Comment:
To have CI properly test this change, you can go to `.travis.yml` and remove the special case in the script/matrix for 1.22.1 and just replace 1.26.1 with it under the `rust` key.

Reply:
Like that? I'm not super familiar with the travis matrix settings.

Reply:
Pretty much, just delete line 19 so it's not adding a redundant build to the matrix.